From: Stephen Hemminger <stephen.hemminger@vyatta.com>
Date: Tue, 5 Feb 2008 11:18:51 +0000 (-0800)
Subject: [NET]: Add if_addrlabel.h to sanitized headers.
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Fstable%2Flinux-2.6.24.y.git;a=commitdiff_plain;h=dded91611a728d65721cdab3dd41d801a356fa15

[NET]: Add if_addrlabel.h to sanitized headers.

if_addrlabel.h is needed for iproute2 usage.

Signed-off-by: Stephen Hemminger <stephen.hemminger@vyatta.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---

diff --git a/include/linux/Kbuild b/include/linux/Kbuild
index c0f9bb7..9363122 100644
--- a/include/linux/Kbuild
+++ b/include/linux/Kbuild
@@ -219,6 +219,7 @@ unifdef-y += i2c-dev.h
 unifdef-y += icmp.h
 unifdef-y += icmpv6.h
 unifdef-y += if_addr.h
+unifdef-y += if_addrlabel.h
 unifdef-y += if_arp.h
 unifdef-y += if_bridge.h
 unifdef-y += if_ec.h
